multiapps-controller-core/src/main/java/org/cloudfoundry/multiapps/controller/core/validators/parameters/FileMimeTypeValidator.java

```diff
@@ -0,0 +1,70 @@
+package org.cloudfoundry.multiapps.controller.core.validators.parameters;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.text.MessageFormat;
+
+import org.apache.commons.io.FilenameUtils;
+import org.apache.tika.Tika;
+import org.cloudfoundry.multiapps.common.SLException;
+import org.cloudfoundry.multiapps.controller.core.Messages;
+import org.springframework.web.multipart.MultipartFile;
+import org.yaml.snakeyaml.Yaml;
+import org.yaml.snakeyaml.error.YAMLException;
+
+public class FileMimeTypeValidator {
+
+    private static final String APPLICATION_ZIP_MIME_TYPE = "application/zip";
+    private static final String APPLICATION_OCTET_STREAM_MIME_TYPE = "application/octet-stream";
+    private static final String TEXT_PLAIN_MIME_TYPE = "text/plain";
+    private static final String YAML_FILE_EXTENSION = "yaml";
+    private static final String EXTENSION_DESCRIPTOR_FILE_EXTENSION = "mtaext";
+    private static final Tika tika = new Tika();
+
+    private FileMimeTypeValidator() {
+    }
+
+    public static void validateMultipartFileMimeType(MultipartFile multipartFile) {
```

Comment:
why is everything static?
This cannot be mocked if needed. Consider using dependency injection intead